src/main/java/edu/ucsb/cs56/ucsb_courses_search/controller/advice/AuthControllerAdvice.java

```diff
@@ -20,13 +20,19 @@ public boolean getIsLoggedIn(OAuth2AuthenticationToken token){
     @ModelAttribute("id")
     public String getUid(OAuth2AuthenticationToken token){
         if (token == null) return "";
-        return token.getPrincipal().getAttributes().get("id").toString();
+        for (java.util.Map.Entry<java.lang.String, java.lang.Object> entry : token.getPrincipal().getAttributes().entrySet()) {
+            System.out.println(entry.getKey() + ":" + entry.getValue().toString());
+        }
+        return token.getPrincipal().getAttributes().get("sub").toString();
```

Comment:
This change needs to be made in `CourseController.java` as well. Accessing the personal schedule in the current state leads to an error page.